bestax-migrate/scripts/validate-corpus-rbx.mjs

```
@@ -0,0 +1,217 @@
+const { runTransform } = await import(distRunner);
+
+/**
+ * Pull the `<Playground>…</Playground>` bodies out of one MDX page. Brace and
+ * tag depth are tracked together so a nested `<Playground>` inside a string or
+ * a render-prop body cannot end the block early.
+ */
+function extractPlaygrounds(source) {
+  const blocks = [];
+  const open = /<Playground>/g;
+  let match;
+  while ((match = open.exec(source))) {
```

ESLint

[error] 'match' is assigned a value but never used.

(no-unused-vars)